### ISSUE TITLE
Steam failure on very recent versions

### ISSUE THREAD

**Issue by cybik** [pop-os/beta#166]
Steam does not start right now, on 21.04.

Libraries that may be of concern, as identified by a command over on https://bugs.archlinux.org/task/71092#comment200392

```
 cybik@hithere > ~/.steam/root/ubuntu12_32 >> trunk ± > file * | grep ELF | cut -d: -f1 | LD_LIBRARY_PATH=. xargs ldd | grep 'not found' | sort | uniq
        libpipewire-0.3.so.0 => not found
        libudev.so.0 => not found
```

In conclusion, packages that possibly need i386 recompiles for steam:latest :
* libpipewire-0.3-0
* libudev0

Related ticket: https://github.com/ValveSoftware/steam-for-linux/issues/7813

Note that this issue may be transient and could be fixed by a Steam update. In the interim, I thought it best to open an issue about it here.

**Comment by mmstick** [comment]
Unfortunately, we can't build i386 packages unless Ubuntu has them on a whitelist for Launchpad PPAs.

**Comment by trgeiger** [comment]
I installed `libnm.so.0` on Fedora and it seems to have fixed the issue, if there's any other Fedora users in here. Got the idea from that Arch link above where it seems that was their solution as well.

**Comment by cybik** [comment]
@trgeiger it worked for *one boot* only on my side. One reboot and it doesn't work anymore. It's really odd.

**Comment by bflanagin** [comment]
Everything is working on my end. @cybik did you upgrade from a previous version or is the is a fresh install? I want to try to replicate the issue as close as I can.

**Comment by cybik** [comment]
@bflanagin yep.

* Recent upgrade from Pop 20.xx to 21.04 Beta
* Had to revert my nVidia drivers from 465 for 18.xx before upgrading
* Had to reinstall a few packages after the distro upgrade
* Until yesterday I had a very "artesan"-type pipewire-based setup overriding 20.xx's pipewire packages with a Debian Experimental `.deb` set.
  * I reverted to hirsute's packages yesterday
  * I've also kept the pulse disablement and pipewire enablement setup

I think that's the whole of my deviations from baseline at this time. Aside from a bunch of ppas.
